### PR TITLE
Reduce subsidy schedule

### PR DESCRIPTION
**Pull request by django92**
Adjust subsidy schedule to account for change in merged mining dynamics.

### CODE REVIEW

**Comment by btcdrak** [comment]
So according to the code from block 2057000 subsidy would begin halving every 6 months.

**Comment by reorder** [comment]
@django92 Please fix unit tests too.

**Comment by muuyuu** [comment]
ACK

**Comment by reorder** [comment]
ACK

**Comment by RedPandaBoy** [comment]
ACK

**Comment by l-crypto** [comment]
ACK :+1:

**Comment by btcexpress** [comment]
ack

**Comment by cryptoworlds** [comment]
ACK as long as auxpow stays enabled

**Comment by TheHulker** [comment]
ACK

File: src/main.cpp [hunk 1]
```diff
@@ -1315,24 +1314,15 @@ CAmount GetBlockValue(int nHeight, const CAmount& nFees)
...
-        // next 4 years
-        nSubsidy = 1 * COIN;
-    } else if (nHeight <= (6 * tHeight)) {
-        // next 4 years
-        nSubsidy = 0.5 * COIN;
+    } else { // (nHeight > 1400000)
+        int halvings = nHeight / Params().SubsidyHalvingInterval();
+        // Force block reward to zero when right shift is undefined.
+        if (halvings <= 64) {
+            nSubsidy = 10 * COIN;
```
Comment (btcdrak) [inline src/main.cpp]: This looks like it needs to be `20 * COIN`

**Comment by reorder** [comment]
@django92 On a closer look the math is off: since 1400000 is not a multiple of 675000, proposed hardfork moment is already back in time and it cannot be carried out. It makes sense to hardfork at block 1400000+675000 (half a year from now) instead. Please adjust the code accordingly.

**Comment by tedunite** [comment]
NACK

I know that auxpow happened, but what is the impact if we switch to the new subsidy patch?
is it all halleluja that we have less inflation? Or is there also a catch if we look further down the road?

IMO we cannot just change the inflation when it looks convenient. I understand that it can give a boost on the short term, but it also will affect the trust on the long term and can destroy Viacoin. 
Please explain why this change is necessary.

**Comment by django92** [comment]
@tedunite The subsidy should have been adjusted at the same time as the auxpow hardfork. There is no downside to changing the subsidy because it wont impact security.

@reorder Fixed.

**Comment by reorder** [comment]
@django92 Hate to say it, but this still seems off: 1971001/657000 == 3, so after block 1971000 subsidy becomes 10 >> 3 == 1.25 instead of 2.5

**Comment by btcdrak** [comment]
Given this is a hard fork, should probably bump the block version and release version.

**Comment by btcdrak** [comment]
Non-blocking concept NACK.

ACK on the technical considerations.

**Comment by btcdrak** [comment]
nit: missing some documentation.

**Comment by btcdrak** [comment]
Might be good to have some unit tests for the hard fork switch too please. Sorry to be a nit picker but have to be careful.

**Pull request merged**